sklearn/datasets/tests/test_common.py

```diff
@@ -16,6 +16,9 @@
         "fetch_openml": pytest.mark.xfail(
             reason="fetch_opeml requires a dataset name or id"
         ),
+        "fetch_lfw_people": pytest.mark.xfail(
+            reason="fetch_lfw_people can fail if pillow is not installed"
+        )
```

Comment:
This is only expect to fail when pillow is not installed. I would prefer a `skipif` when pillow is not installed

```py
try:
    import PIL
    pillow_installed = True
except ImportError:
    pillow_installed = False
```

```suggestion
        "fetch_lfw_people": pytest.mark.skipif(not pillow_installed
            reason="pillow not installed"
        )
```

Although this means that `KNOWN_FAILURE ` would have to be renamed to maybe `FETCH_MARKS`. WDYT  @glemaitre ?

Reply:
+1 about this solution.